tutorials/tutorial01_sumo.ipynb

```diff
@@ -338,11 +338,25 @@
     "# create the environment object\n",
     "env = AccelEnv(env_params, sumo_params, network)\n",
     "\n",
-    "# create the experiment object\n",
-    "exp = Experiment(env)\n",
+    "flow_params = dict(\n",
```

Comment:
can you add the descriptions to each parameter the way we do in the examples?

Reply:
Actually I removed the descriptions for 2 reasons: 1- It would make the tutorial kind of long. I realized most of code snippets in the tutorials are short, not to distract the reader. 2- Those parameters are all explained before in the tutorial.
@AboudyKreidieh I am open to discussing. What do you think?

Reply:
sounds good to me. So long as we agree it makes the tutorials clearer